app/src/main/java/woowacourse/paint/model/brush/Circle.kt

```diff
@@ -0,0 +1,45 @@
+package woowacourse.paint.model.brush
+
+import android.graphics.Paint
+import android.graphics.Path
+
+object Circle : Brush() {
+    private var beforePosition = Pair(0f, 0f)
+    override fun setStyle() {
+        paintInstance.style = Paint.Style.FILL
+    }
+
+    fun setCurrentPosition(xCursor: Float, yCursor: Float) {
+        beforePosition = Pair(xCursor, yCursor)
+    }
+
+    fun drawPreview(xCursor: Float, yCursor: Float) {
+        previewDraw = Pair(
+            Path().apply {
+                addCircle(
+                    beforePosition.first + ((xCursor - beforePosition.first) / 2),
+                    beforePosition.second + ((xCursor - beforePosition.first) / 2),
+                    (xCursor - beforePosition.first) / 2,
+                    Path.Direction.CW,
+                )
+            },
+            Paint().apply { set(paintInstance) },
+        )
```

Comment:
```suggestion
        val path = Path().apply {
            addCircle(
                beforePosition.first + ((xCursor - beforePosition.first) / 2),
                beforePosition.second + ((xCursor - beforePosition.first) / 2),
                (xCursor - beforePosition.first) / 2,
                Path.Direction.CW,
            )
        }
        val paint = Paint().apply { set(paintInstance) }

        previewDraw = path to paint
```
어떤 코드가 로직을 더 파악하기 쉽나요? 궁금해서 여쭤봅니다 😊

Reply:
각각 만드는게 확실히 낫네요! 최곱니다👍